cmd/influxd/launcher/launcher.go

```diff
@@ -101,7 +100,7 @@ const (
 	MaxInt = 1<<uint(strconv.IntSize-1) - 1
 )
 
-func NewInfluxdCommand(ctx context.Context, subCommands ...*cobra.Command) *cobra.Command {
```

Comment:
The humble beginnings of my quest to simplify `launcher.go`...

Reply:
And a noble quest it is!